install-deps.sh

```diff
@@ -31,6 +31,8 @@ if [ x`uname`x = xFreeBSDx ]; then
         devel/gperf \
         devel/gmake \
         devel/cmake \
+        devel/cunit \
+        devel/openssl \
```

Comment:
@liupan1111 
`(open)ssl` is not required for FreeBSD since the base-install already had SSL onboard.
Only if you need really special things from the newest version this is